src/NHibernate/Multi/QueryBatchItemBase.cs

```diff
@@ -91,9 +92,8 @@ protected class QueryInfo : ICachingInformation, ICachingInformationWithFetches
 			/// <param name="loader">The loader.</param>
 			/// <param name="querySpaces">The query spaces.</param>
 			/// <param name="session">The session of the query.</param>
-			public QueryInfo(
-				QueryParameters parameters, Loader.Loader loader, ISet<string> querySpaces,
-				ISessionImplementor session)
+			public QueryInfo(QueryParameters parameters, ILoader loader, ISet<string> querySpaces,
+			                 ISessionImplementor session)
```

Comment:
Revert the formatting.

(I have ceased flagging binary breaking changes, it seems this PR will not avoid them and will have to be merged only for 6.0.)

Reply:
Possible solution without breaking it.

```suggestion
		
			// 6.0 TODO : remove
			
			/// <summary>
			/// Create a new <c>QueryInfo</c>.
			/// </summary>
			/// <param name="parameters">The query parameters.</param>
			/// <param name="loader">The loader.</param>
			/// <param name="querySpaces">The query spaces.</param>
			/// <param name="session">The session of the query.</param>
			public QueryInfo(
				QueryParameters parameters, Loader.Loader loader, ISet<string> querySpaces,
				ISessionImplementor session)
				:this(parameters, (ILoader)loader, querySpaces, session)
			{
				
			}

			/// <summary>
			/// Create a new <c>QueryInfo</c>.
			/// </summary>
			/// <param name="parameters">The query parameters.</param>
			/// <param name="loader">The loader.</param>
			/// <param name="querySpaces">The query spaces.</param>
			/// <param name="session">The session of the query.</param>
			public QueryInfo(
				QueryParameters parameters, ILoader loader, ISet<string> querySpaces,
				ISessionImplementor session)
			{
				Parameters = parameters;
				Loader = loader;
				QuerySpaces = querySpaces;

				IsCacheable = loader.IsCacheable(parameters);
				if (!IsCacheable)
					return;

				CacheKey = Loader.GenerateQueryKey(session, Parameters);
				CanGetFromCache = Parameters.CanGetFromCache(session);
				CanPutToCache = Parameters.CanPutToCache(session);
			}
```

Reply:
Ok.